### ISSUE TITLE
Retorno envio provedor Equiplano

### ISSUE THREAD

**Issue by flavionootech** [OpenAC-Net/OpenAC.Net.NFSe#26]
Não esta carregando os erros próprios do retorno XML do equiplano.
Segue xml de retorno:

<ns:return xmlns:ns="http://services.enfsws.es">&lt;?xml version="1.0" encoding="UTF-8"?&gt;

&lt;es:esEnviarLoteRpsResposta xmlns:es="http://www.equiplano.com.br/esnfs" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.equiplano.com.br/esnfs enviarLoteRpsResposta_v01.xsd"&gt;

  &lt;mensagemRetorno&gt;

    &lt;listaErros&gt;

      &lt;erro&gt;

        &lt;cdMensagem&gt;8011&lt;/cdMensagem&gt;

        &lt;dsMensagem&gt;O RPS informado já foi convertido em NFS. Utilize o serviço esConsultarNfsePorRps antes de enviar o RPS.&lt;/dsMensagem&gt;

      &lt;/erro&gt;

    &lt;/listaErros&gt;

  &lt;/mensagemRetorno&gt;

&lt;/es:esEnviarLoteRpsResposta&gt;

</ns:return>

**Comment by rftd** [comment]
ai ai como sempre os provedores fazendo caca

**Comment by flavionootech** [comment]
Peguei o fonte, fiz alteração e tentei colocar a referencia direto do assembly, porém da erro que não pode criar canal seguro TLS, sabe dizer o que pode ser?

**Comment by flavionootech** [comment]
Eu vi que tem tratamento para este retorno mas não consegui entender porque não funciona

**Comment by rftd** [comment]
Erro de tls geralmente tem haver com o protocolo que tem que ser usado na comunicação.

**Comment by flavionootech** [comment]
> Erro de tls geralmente tem haver com o protocolo que tem que ser usado na comunicação.

Sim, porém o estranho é que pelo nuget funciona (porém da o problema inicial) se compilar o projeto e inserir manual a referência ocorre este problema

**Comment by flavionootech** [comment]
Descobri que o problema do tls esta sendo causado por uma alteracao no NFSeServiceClient que passava o certificado e agora passa null

**Comment by angelomachado** [comment]
Opa. Recentemente fiz algumas correções no provedor, pois teve algumas alterações nos schemas da Equiplano. O pacote nuget ainda não contém as alterações. Pode testar as dlls compilando o projeto?

**Comment by angelomachado** [comment]
O erro inicial da issue era causado [nessa](https://github.com/OpenAC-Net/OpenAC.Net.NFSe/blob/main/src/OpenAC.Net.NFSe/Providers/Equiplano/EquiplanoServiceClient.cs#L151) linha e foi corrigido.
